src/Bellhop.js

```diff
@@ -99,6 +109,12 @@ export class Bellhop extends BellhopEventDispatcher {
       return;
     }
 
+    if (this.debug && typeof this.debug === 'function') {
```

Comment:
Maybe extract this to a function call? 
Just in case there's an update later, I noticed it's duplicated down below.